### ISSUE TITLE
The specification of arguments as a string has been deprecated

### ISSUE THREAD

**Issue by ealeksandrov** [leshill/open_pod_bay#5]
Appeared after updating to CocoaPods 0.33.0 (maybe earlier, just noticed):

```
[!] The specification of arguments as a string has been deprecated Pod::Command::Open: `[-a]`
```

**Comment by leshill** [comment]
Thanks for the heads up, will fix… or Pull Request?

**Comment by leshill** [comment]
Fixed by #6
